### PR TITLE
Responsive design

### LINKED ISSUE #20: Add responsive design

### LINKED ISSUE #53: Favicon in index.html

### PR DESCRIPTION
**Pull request by Kanderson58**
# Description

- Add media queries to make each page responsive
- Alter favicon for webpage tab

## Type Of Change

- [ ]  Bug fix
- [x]  New feature
- [x]  Refactoring
- [ ]  Testing

# How Has This Been Tested?

- [x]  local host
- [ ]  dev tools
- [ ]  cypress

# Related Issues:

- [x]  closes  #53 closes #20 

# Checklist:

- [x]  My code follows the Turing's guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  No console error messages

### CODE REVIEW

File: src/Components/Dashboard/Dashboard.css
Comment (LeftyLincoln) [inline src/Components/Dashboard/Dashboard.css:1]: Great work on the dashboard responsiveness, I like how it changes the cards.

**Review by LeftyLincoln — APPROVED** [review]
Looks great to me!!!

File: public/index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
     />
     <link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+    <link rel="icon" href="http://example.com/favicon.png">
```
Comment (qrispi) [inline public/index.html:12]: Did you get the favicon working??

File: src/Components/ChoreCard/ChoreCard.css [hunk 2]
```diff
@@ -1,8 +1,8 @@
 .chore-card {
   border: 2px solid black;
-  margin: 1% 3%;
-  padding: 2%;
-  width: 250px;
+  margin: 1%;
+  padding: 1%;
+  width: 240px;
```
Comment (qrispi) [inline src/Components/ChoreCard/ChoreCard.css:5]: It's all in the details :D

File: src/Components/ChoreForm/ChoreForm.css
Comment (qrispi) [inline src/Components/ChoreForm/ChoreForm.css:1]: Nice work adding queries for all the break points!

**Review by qrispi — APPROVED** [review]
LGTM! Thanks for tackling this Kara!

**Pull request merged**